homeassistant/components/knx/websocket.py

```diff
@@ -541,3 +546,115 @@ def ws_create_device(
         configuration_url=f"homeassistant://knx/entities/view?device_id={_device.id}",
     )
     connection.send_result(msg["id"], _device.dict_repr)
+
+
+@websocket_api.require_admin
+@websocket_api.websocket_command(
+    {
+        vol.Required("type"): "knx/get_entity_schemas",
+    }
+)
+@provide_knx
+@callback
+def ws_get_entity_schemas(
```

Comment:
```suggestion
@websocket_api.websocket_command(
    {
        vol.Required("type"): "knx/get_entity_schemas",
        vol.Required("platform"): str,
    }
)
@provide_knx
@callback
def ws_get_entity_schemas(
```
Should this be split up per platform? We probably only need one schema at a given time in the frontend.

Reply:
Since the schemas are static, the idea is to prefetch all schemas when the UI loads, instead of fetching them repeatedly. Additionally, for the initial step of the entity creation process, where the entity type is selected, a list of all supported platforms is required. This is why I created the root node PlatformConfigSchema. If any additional UI configuration parameters are needed at the platform level, this would be the place to add them.

Reply:
We do the same with project file data. It gets fetched when first needed, then sourced from a cache. 
Same could be done for the schema per platform on demand. But that can also be changed later on - when we can see exactly how big that payload gets with multiple platforms.